### PR TITLE
trafficmonitor-lite: Add version 1.83

### LINKED ISSUE #7956: [Request] TrafficMonitor - Lite Version
### Information

**Name:** 
TrafficMonitor - Lite Version

**Description:**
Lite version of TrafficMonitor.  Does not required Administrator to run, does not contain GPU, Hard Disk or temperature monitoring

**Homepage:**
https://github.com/zhongyang219/TrafficMonitor

**Download link(s):** 
https://github.com/zhongyang219/TrafficMonitor/releases

**Some indication of popularity/repute:** 
TrafficMonitor full version is already in extras bucket. 
 https://github.com/ScoopInstaller/Extras/blob/master/bucket/trafficmonitor.json

### PR DESCRIPTION
**Pull request by bin456789**
Closes #7956 

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/trafficmonitor-lite.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+    "version": "1.83",
+    "description": "Network monitoring suspension window software in Windows platform. It can display the current network speed, CPU and memory usage. It also support the functions of display in the taskbar, change skin and historical traffic statistics. (Lite version, does not required Administrator to run, does not contain GPU, Hard Disk or temperature monitoring)",
+    "homepage": "https://github.com/zhongyang219/TrafficMonitor",
+    "license": {
+        "identifier": "Anti 996-License-1.0",
+        "url": "https://github.com/zhongyang219/TrafficMonitor/blob/master/LICENSE"
+    },
```
Comment (rashil2000) [inline bucket/trafficmonitor-lite.json:8]:
```suggestion
    "homepage": "https://github.com/zhongyang219/TrafficMonitor/blob/master/README_en-us.md",
    "license": "MIT",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by rashil2000** [comment]
Do both TrafficMonitor and TrafficMonitor-Lite store their settings in $env:APPDATA\TrafficMonitor folder?

**Comment by bin456789** [comment]
> Do both TrafficMonitor and TrafficMonitor-Lite store their settings in $env:APPDATA\TrafficMonitor folder?

Yes. share same settings in same appdata folder.
but portable is by default

File: bucket/trafficmonitor-lite.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            "TrafficMonitor (Lite)"
+        ]
+    ],
+    "persist": [
+        "config.ini",
+        "global_cfg.ini",
+        "history_traffic.dat",
+        "plugins"
+    ],
+    "checkver": "github",
```
Comment (bin456789) [inline bucket/trafficmonitor-lite.json:41]:
```suggestion
    "checkver": {
        "github": "https://github.com/zhongyang219/TrafficMonitor"
    },
```

**Pull request merged**